### ISSUE TITLE
[COMPAT] [FORGE] [1.20.1] Panda's Falling Trees

### ISSUE THREAD

**Issue by Camawama** [hammertater/treechop#233]
Your mod makes it so the player can slowly chop away at a tree until it breaks. Panda's Falling Trees makes the tree fall after breaking.

Having the two work together would be one step closer to a realistic tree chopping mechanic and I would love to see it work!

I want to be able to chop the bottom of a tree slowly and then, at a certain point, have the tree fall over. 😄

(Currently, when chopping a tree, it ignores TreeChop and just has Panda's mechanics).

https://github.com/PandaDap2006/Pandas-Falling-Trees/issues/37

**Comment by hammertater** [comment]
Yes, I'd love to see this compatibility happen! A while ago I saw a comment by Panda saying they might look into it at some point but for now were focusing on getting the mod more stable/complete/etc. TreeChop's API supports this kind of compatibility, it's just waiting for Panda to make use of it.

**Comment by Camawama** [comment]
Looks like Panda needs some help on the other issue report!
